package.json

```diff
@@ -1,7 +1,8 @@
 {
   "name": "react-firestore",
   "version": "0.0.0-semantically-released",
-  "description": "React components to fetch data from firestore using render props",
+  "description":
```

Comment:
@peternoordijk - Overall, it seems like not much changed in this file but the formatting got changed a decent amount. Do you have a formatter set up?

Reply:
Ah-- yes, I do. It's just VSCode with format on save. I guess I should've turned that off.

Reply:
@green-arrow just to be clear: do you want me to change this back? 😄

Reply:
@peternoordijk - Yeah, that would be awesome 😄 Thanks!

Reply:
@green-arrow no problem :) By the way I remember now what caused this change: yarn was doing all this formatting automatically 🙄

Reply:
@green-arrow Hold on! This formatting is not done by yarn! This formatting happens in some git pre commit hook. Does that by any chance sound familiar to you? I'm very sure it happens in there because I was fixing the formatting using Vim this time, and just before committing it's changing right back.

Reply:
@peternoordijk - no worries then! Thanks for looking into it!